### PR TITLE
Create 2021-10-19-Subnero-Vortice-partnership.md

### PR DESCRIPTION
**Pull request by Florie25**

### CODE REVIEW

File: _posts/2021-10-19-Subnero-Vortice-partnership.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: New business partner for Subnero in Republic of Korea
```
Comment (AnneThKu) [inline _posts/2021-10-19-Subnero-Vortice-partnership.md]: Is this partner in Korea or Portugal?

File: _posts/2021-10-19-Subnero-Vortice-partnership.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-08
```
Comment (AnneThKu) [inline _posts/2021-10-19-Subnero-Vortice-partnership.md]: Is the date supposed to be backdated to 8th of July?

**Comment by AnneThKu** [comment]
Please find below the screenshots.

Screenshot 2021-11-17 at 11 04 56 AM

Screenshot 2021-11-17 at 11 06 23 AM

File: _posts/2021-10-19-Subnero-Vortice-partnership.md (hunk 1 above)
Comment (Florie25) [reply _posts/2021-10-19-Subnero-Vortice-partnership.md]: Well spotted. Thanks

File: _posts/2021-10-19-Subnero-Vortice-partnership.md (hunk 2 above)
Comment (Florie25) [reply _posts/2021-10-19-Subnero-Vortice-partnership.md]: Again, thank you for having an eye for details.

**Review by AnneThKu — DISMISSED** [review]

**Review by notthetup — DISMISSED** [review]

File: _posts/2021-10-19-Subnero-Vortice-partnership.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: New business partner for Subnero in Portugal
+banner : images/banner-pulse-standard.jpg
+date:  2021-10-19
+categories: news wnc
+thumbnail: images/pulse-vortice.jpg
+---
+
+In its quest of expansion throughout Europe, Subnero has appointed [Vortice](https://vortice-lda.pt/){:target="_blank"} as its new business partner in Portugal.
```
Comment (manuignatius) [inline _posts/2021-10-19-Subnero-Vortice-partnership.md]:
```suggestion
In our quest for expansion throughout Europe, Subnero has appointed [Vortice](https://vortice-lda.pt/){:target="_blank"} as its new business partner in Portugal.
```

**Review by manuignatius — DISMISSED** [review]

**Comment by manuignatius** [comment]
There is one comment, please address it before merging it.

**Review by notthetup — APPROVED** [review]

**Review by AnneThKu — APPROVED** [review]

**Review by manuignatius — APPROVED** [review]

**Pull request merged**